### PR TITLE
Added 1.21 support + Fixed 1.20.5/6 support

### PR DESCRIPTION
**Pull request by Lagggpixel**

### CODE REVIEW

File: versionsupport_v1_21_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_21_R1/v1_21_R1.java [hunk 1]
```diff
@@ -685,9 +690,9 @@ public String getMainLevel() {
     @Override
     public Fireball setFireballDirection(Fireball fireball, Vector vector) {
         EntityFireball fb = ((CraftFireball) fireball).getHandle();
-        fb.d = vector.getX() * 0.1D;
-        fb.e = vector.getY() * 0.1D;
-        fb.f = vector.getZ() * 0.1D;
+        // fb.d = vector.getX() * 0.1D;
```
Comment (Lagggpixel) [inline versionsupport_v1_21_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_21_R1/v1_21_R1.java]: Further testing is needed here

Comment (tomkeuper) [reply versionsupport_v1_21_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_21_R1/v1_21_R1.java]: https://mappings.dev/1.21/net/minecraft/world/entity/projectile/AbstractHurtingProjectile.html

**Comment by ritana5** [comment]
Theoretically if 1.20.5 is supported it should work on 1.20.6?

**Comment by tomkeuper** [comment]
> Theoretically if 1.20.5 is supported it should work on 1.20.6?

As far as I can see 1.20.6 is not fully compatible with 1.20.6 since there seem to be some differences for holograms

**Comment by tomkeuper** [comment]
Or I might still need to double check if holograms work correctly on 1.20.5

**Comment by tomkeuper** [comment]
Support for 1.20.6 added aswel!

**Pull request merged**